index.rst

```diff
@@ -10,20 +10,849 @@
+
+- DM tooling must be installed and maintained on the Camera Diagnostic Cluster
+- Rapid-analysis-critical must be developed and deployed, with the ability to only focus on a subset of detectors, and/or metrics
+- The database containing the output must be hosted on the summit, then replicated outwards
+
+Note that the full output of Rapid Analysis cannot be computed due to the limited compute power.
+
+This committee suggests that if Antu does need to stay at the base, then a step-wise approach where the infrastructure for scenario 1 gets implemented prior to significant effort being put into scenario 2, if deemed appropriate.
+The preferred solution is to move the Antu servers to the summit.
+
+.. _antu_at_summit:
+
+Antu at the Summit (Proposed Change)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another possibility which has been considered by this group is to relocate the Antu servers to the summit, even if it means reducing the cluster size in Chile and increasing the capability at the USDF.
+This scenario reduces the scope of the commissioning cluster, essentially relocating the functionality of a general compute facility to the USDF, and having the cluster be a more direct support to on-the-fly observations and reductions.
+In doing so, this allows States 1 and 2 above to be supported when a network outage to the summit occurs.
+Furthermore, it simplifies the number of systems that require support which significantly reduces the workload of the IT group.
+
+The first hurdle of moving Antu to the summit is the capacity to store, power, and cool the servers.
+The Chilean IT manager, Christian Silva, informed us that 2500 cores can be run on Cerro Pachón if needed.
+However, the support is based around 22 nodes or ~1400 cores, which is Yagan (being upgraded to 640 cores) and Antu (784 cores).
+Therefore, capacity is not an issue.
+However, we must also consider what computing resources are required to support the two main use-cases for Antu:
+
+1. Running Rapid Analysis and the necessary display tools
+2. Being able to run full-focal plane wavefront sensing by pistoning the entire camera in and out of focus
+
+FAFF has shown that item 1 is feasible, which was presented in the `Potential Paths for Implementation`_ subsection of `Deliverable 2: Rapid Analysis Calculated Metrics`_, albeit with a limited number of detectors.
+The full focal plane sensing use-case suffers the same limitations of the Rapid Analysis framework, and has an increased computational load.
+Currently, the full analysis takes approximately 3 minutes using 2-cores per chip on Antu, and is independent of location.
+However, moving the Antu servers to the summit enables this processing to occur in the event of an outage to the base.
```

Comment:
So we discussed in the meeting (maybe you weren't there?) that we just want to say we're moving the servers because they might just get incorporated into Yagan and Antu would disappear.